webapp/src/components/Chip/Chip.js

```diff
@@ -2,62 +2,59 @@ import React from 'react'
 import PropTypes from 'prop-types'
 import { Icon } from 'semantic-ui-react'
 
-import { parcelType } from 'components/types'
-import './ParcelCoord.css'
+import './Chip.css'
 
-export default class ParcelCoord extends React.PureComponent {
+export default class Chip extends React.PureComponent {
   static propTypes = {
+    className: PropTypes.string,
     size: PropTypes.oneOf(['small', 'medium', 'large']),
-    parcel: parcelType,
     status: PropTypes.string,
+    children: PropTypes.node,
     onClick: PropTypes.func,
     onDelete: PropTypes.func
   }
 
   static defaultProps = {
+    className: '',
     size: 'medium',
     status: ''
   }
 
   handleOnClick = () => {
-    const { onClick, parcel } = this.props
-    if (onClick) {
-      onClick(parcel)
-    }
+    const { onClick } = this.props
+    if (onClick) onClick()
   }
 
   handleOnDelete = event => {
-    const { onDelete, parcel } = this.props
-    onDelete(parcel)
-    event.stopPropagation()
-    event.nativeEvent.stopImmediatePropagation()
+    const { onDelete } = this.props
+    if (onDelete) {
+      onDelete()
+      event.stopPropagation()
+      event.nativeEvent.stopImmediatePropagation()
+    }
   }
 
   getClassName() {
-    const { size, status, onClick, onDelete } = this.props
+    const { className, size, status, onClick, onDelete } = this.props
     const statusClass = status
       ? `has-status status-${status.toLowerCase()}`
       : ''
     const onClickClass = onClick ? 'clickeable' : ''
     const onDeleteClass = onDelete ? 'deleteable' : ''
-    return `ParcelCoord ${size} ${statusClass} ${onClickClass} ${onDeleteClass}`
+    return `Chip ${className} ${size} ${statusClass} ${onClickClass} ${onDeleteClass}`
```

Comment:
Just to be consistent with the rest of the variables. What do you think of adding

```javascript
const sizeClass = size
```